setup.cfg

```diff
@@ -0,0 +1,8 @@
+[flake8]
+max-line-length = 95
```

Comment:
**THAT** is 100% and totally at the discretion of the project BDFL (https://en.wikipedia.org/wiki/Benevolent_dictator_for_life). Line length is something that people get religious about and I don't want to spend 2 seconds discussing it. I wrote 95 here, because in other projects we have been going with recommendations that have been mentioned in a few different places, which is aim for 90 but allow up to 95, which essentially means that the developer should break at 90 unless there is a good reason, like a long link or something like that, which can be finished in the last 5. But since linters aren't that intelligent, it needs to allow up to 95 if that is the policy. But I will put there whatever you say.